sdk/python/feast/infra/offline_stores/file.py

```diff
@@ -599,6 +599,11 @@ def _normalize_timestamp(
         created_timestamp_column_type = df_to_join_types[created_timestamp_column]
 
     if not hasattr(timestamp_field_type, "tz") or timestamp_field_type.tz != pytz.UTC:
+        # if you are querying for the event timestamp field, we have to deduplicate
```

Comment:
not fully sure i understand what these duplicate columns are coming from? Can you leave more comments on why?

Reply:
yeah i'll update the PR but basically querying for `created` in `get_online_features` duplicates the `created_timestamp_column`. So I could go deeper and try to fix the duplication but I needed a quick fix on my side and it works correctly for us.

Reply:
@adchia updated the comment 👍